Table view

yyon opened the pull request.

gjw32 commented:
- Table mode is not working in Excel
- On HTML tables in browsers, each column header cell has "column header" attached to the end of the text. Consider stripping this out as extraneous.
- Focus should follow NVDA focus, so if NVDA leaves a table, exit table mode. If focus in a table moves beyond what is visible on the display, the display should scroll accordingly

yyon commented:
@gjw32 Does excel not have its table in a "table" role in the accessibility tree? If not, what role is it? I don't own excel so I can't test. 
I changed the following focus and removed the "column header" text as you suggested

yyon commented:
@gjw32 It looks like you pressed alt+f4 and then the log is immediately followed by a bunch of errors. Does the problem occur after closing a window? Was excel the window that was focused after closing the window? 

Also, in NVDA advanced settings, what is the "use UI automation to access microsoft excel spreadsheet controls when available" option set to? I have it off.

yyon commented:
@gjw32 I'm pretty sure this should fix the freezing issue - try this
